core/primitives/src/views.rs

```diff
@@ -1179,6 +1182,12 @@ pub enum ActionView {
         #[serde_as(as = "Base64")]
         code: Vec<u8>,
     },
+    UseGlobalContract {
+        code_hash: CryptoHash,
+    },
+    UseGlobalContractByAccountId {
```

Comment:
Checking if it's fine to have two separate enums here while we have just one in `Action::UseGlobalContract`?

Reply:
In my understanding it is OK, since `Action` is our internal representation while `ActionView` is public API. So I decided to have single `Action` because it is more concise. Having separate `ActionView` for deploying by account id avoids overcomplicating public api with choosing the identifier.